src/greenlight/runner.clj

```diff
@@ -90,10 +177,12 @@
      (let [system (component/start (new-system))]
        (try
          (binding [test/*report* (partial report/handle-test-event
-                                         {:print-color (not (:no-color options))})]
+                                          {:print-color (not (:no-color options))})]
            (println "Running" (count tests) "tests...")
-           (let [results (mapv (partial test/run-test! system) tests)]
-             ; TODO: check result spec?
+           (let [results (if-let [n-threads (:parallel options)]
```

Comment:
An idea: you could set a `:default 1` in the options definition, then test that it's greater than one before invoking the parallel path. You should definitely validate somewhere that if the option is set, it's a positive integer.